La-Mula-App-main/src/main/kotlin/com/lamulaapp/domain/Orders.kt

```diff
@@ -0,0 +1,39 @@
+package com.lamulaapp.domain
+
+import jakarta.persistence.*
+import java.io.Serial
+import java.util.*
+
+data class Orders(
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    val id: UUID? = null,
+
+    @OneToMany(cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
+    @JoinColumn(name = "id_user", referencedColumnName = "id_user")
+    val iduser: Users? = null,
+
+    @OneToMany(cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
+    @JoinColumn(name = "id_oder_status", referencedColumnName = "id_order_status")
+    val idorderstatus: Order_Status? = null,
+
+    val ordercode: Serial,

```

Comment:
debes agregar la linea en ese serial por que es un codigo de referencia.

```suggestion
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val ordercode: Serial,

```